fabbwled-backend/src/test/java/ch/bbw/fabbwled/lands/CliTestMain.java

```diff
@@ -0,0 +1,76 @@
+package ch.bbw.fabbwled.lands;
+
+import ch.bbw.fabbwled.lands.book.SectionDto;
+import ch.bbw.fabbwled.lands.book.SectionId;
+import ch.bbw.fabbwled.lands.controller.PlayerController;
+import ch.bbw.fabbwled.lands.controller.SectionController;
+import ch.bbw.fabbwled.lands.service.PlayerSession;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+
+import javax.swing.*;
+import java.awt.*;
+import java.util.stream.Stream;
+
+import static org.junit.jupiter.api.Assumptions.assumeTrue;
+
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.MOCK, properties = {"logging.level.ch.bbw.fabbwled=WARN"})
+@ActiveProfiles("test")
+class CliTestMain {
+
+	@Autowired
+	PlayerSession playerSession;
+
+	@Autowired
+	PlayerController playerController;
+
+	@Autowired
+	SectionController sectionController;
+
+	@BeforeAll
+	static void onlyInHeadless() {
+		assumeTrue(!GraphicsEnvironment.isHeadless());
+        assumeTrue(System.getenv("CI") == null);
+	}
+
+	@Test
+	void run() {
```

Comment:
does java/gradle have some way to add additional main programs so that we don't have to hack this as a test like this?